style.css

```diff
@@ -0,0 +1,74 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 16px 24px;
+}
+
+.logo {
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;

```

Comment:
- [ ] [REQUIRED] For increasing 🆙 your chances in the job search 💼 after finishing microverse, we request that you follow coding best practices 💯  in all of your projects. One of these practices is **DRY** (**D**on't **R**epeact **Y**ourself). That means if you find yourself repeating 🔁 a specific line of code in multiple places, you have to find a way to only use it once 1️⃣. This way when you update ♻️  this line, you won't have to remember where else did you write that to update the other copies.

### You can check this in the [Html & Css Best Practices Page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

![image](https://user-images.githubusercontent.com/60979926/164267978-27c9c5e1-37bb-4d2d-81ef-e7cd491bfbdb.png)

### In our case

you can easily use a `.font` class in multiple places or simply put the `font-family` property in the `body` selector and all other children elements going to inherit it by default.

- (your code)

```css
.logo {
  /* ...code */
  font-family: 'Poppins', sans-serif;
  /* ...code */
}

.primary-heading {
  /* ...code */
  font-family: 'Poppins', sans-serif;
  /* ...code */
}
```

- suggestion

```css
.font {
  font-family: 'Poppins', sans-serif;
}
```